### DISCUSSION TITLE
Is it possible to compile against nvim 0.8.0-dev?

### DISCUSSION THREAD

**Discussion by qianwch** [qvacua/vimr#956] [Q&A] [answered]
I need the vim.lsp.buf.format() function only available in 0.8.0

**Comment by qvacua** [comment, marked as answer]
Thanks to neovim/neovim@d0644fa9bfc3fa3abdd886d2ef13533778317154, it's now much easier to build universal binary. I'll try to set up a nightly build.

-> https://github.com/qvacua/vimr/releases/tag/neovim-nightly
